### PR TITLE
changes made to include component name in GitHub link

### PR DESCRIPTION
**Pull request by ksanctis**
https://rally1.rallydev.com/#/15545167705d/dashboard?detail=%2Fdefect%2F641821272359

### CODE REVIEW

File: tools/eleventy-utils.cjs [hunk 1]
```diff
@@ -58,10 +58,20 @@ module.exports = {
...
+		let markdownDirectory = devMarkdown; // e.g., components/buttons/README.md
+		if (codeDirectory) {
+			markdownDirectory = codeDirectory;
+		} else if (devMarkdown && devMarkdown.includes('/')) {
+			const markdownMatch = devMarkdown.match(/(.*)\/.*.md/); // match would be components/buttons
+			if (markdownMatch && markdownMatch.length === 2) markdownDirectory = markdownMatch[1];
+		} else {
+			markdownDirectory = ''; // case where devMarkdown is just README.md
+		}
+		if (markdownDirectory !== '') return `${repoName}${markdownDirectory}`;
```
[the remark this answers is not in the document]
Comment (ksanctis) [reply tools/eleventy-utils.cjs]: `markdownDirectory` should now be cleaned up and there are checks and results for when `repoName` ends with a `/ `or not. The cleanup of `markdownDirectory` should results in it always not starting with a `/`.

[the remark this answers is not in the document]
Comment (ksanctis) [reply tools/eleventy-utils.cjs]: The cleanup of `markdownDirectory` should result in it always not starting with a `/`.

File: tools/eleventy-utils.cjs [hunk 2]
```diff
@@ -58,11 +58,22 @@ module.exports = {
...
+		let markdownDirectory = devMarkdown; // e.g., components/buttons/README.md
+		if (codeDirectory) {
+			markdownDirectory = codeDirectory;
+		} else if (devMarkdown && devMarkdown.includes('/')) {
+			const markdownMatch = devMarkdown.match(/(.*)\/.*.md/); // match would be components/buttons
+			if (markdownMatch && markdownMatch.length === 2) markdownDirectory = markdownMatch[1];
+		} else {
+			markdownDirectory = ''; // case where devMarkdown is just README.md
+		}
+		markdownDirectory = markdownDirectory.replace(/^\//g, ''); // remove extra backslashes
```
Comment (oelhanafey) [inline tools/eleventy-utils.cjs:74]:
```suggestion
		markdownDirectory = markdownDirectory.replace(/^\//g, ''); // remove starting backslash
```

Nit: this comment isn't correct anymore after changing the regex

**Review by oelhanafey — APPROVED** [review]
Nice job!

**Pull request merged**